less/components/_commons.less

```diff
@@ -188,4 +191,12 @@ footer#solstice-footer {
 
 .product-list .grid-center { 
     display: flex; align-items: center;
-}
\ No newline at end of file
+}
+.featured-section-members {
+    .owl-item {
+        display: flex;
+        justify-content: flex-end;
+        flex-direction: column;
+        height:115px;
```

Comment:
our style guide suggests spaces between the colon and the value. There are a few other instances of this, but we can put this off until your final cleanup of the CSS if you like